src/blenderbim/blenderbim/bim/module/pset/ui.py

```diff
@@ -139,6 +141,11 @@ def draw_psetqto_ui(context, pset_id, pset, props, layout, obj_type):
 def draw_psetqto_editable_ui(box, props, prop):
     row = box.row(align=True)
     draw_property(prop, row, copy_operator="bim.copy_property_to_selection")
+    obj = bpy.context.active_object
```

Comment:
Don't check this ins the draw function, since checking what is going to be a pretty big dictionary on every draw call will be expensive. Instead modify this to precalculate it in the data/import function (I haven't checked exactly where).

Also note that not all properties will have an active object (e.g. task, cost, or resource properties) so check the type of property before doing this.

Reply:
@Moult  Yeah, i forgot about the data thing, sorry.
Is it better now?